test/unit/models/events.test.js

```diff
@@ -74,4 +76,117 @@ describe("Events", function () {
       }
     });
   });
+
```

Comment:
nit: can we have have test coverage for `controllers/events.js` and `models/event.js` to be at least 85%?

Reply:
https://github.com/Real-Dev-Squad/website-backend/pull/1165#discussion_r1238036054